src/services/etherspot.js

```diff
@@ -600,6 +618,16 @@ export class EtherspotService {
       return null;
     }
 
+    if (chain === CHAIN.AVALANCHE) {
+      return sdk
+        .getTransactions({ account: sdk.state.accountAddress })
+        .then(({ items }) => items)
+        .catch((error) => {
+          reportErrorLog('getTransactionsByAddress -> getTransactions failed', { address, chain, error });
+          return null;
+        });
+    }
+
     return sdk
       .getTransactions({ account: address })
```

Comment:
Do you think it's better to change this line to `.getTransactions({ account: sdk.state.accountAddress })` as opposed to creating a `if` statement just for the one chain above on line 621?

Reply:
yeah it is